mod_ci/controllers.py

```diff
@@ -726,7 +729,8 @@ def progress_reporter(test_id, token):
 
                 if status == TestStatus.canceled:
                     state = Status.ERROR
-                    message = 'Tests aborted due to an error; please check'
+                    error_id = TestResult.regression_test_id
+                    message = 'Tests aborted due to the following error_id error: {}'.format(error_id)
```

Comment:
This is not as per the requirements:
- This will only be posted when the tests are aborted or canceled, so never when tests complete, but fail
- It will only post for a single test that failed

In short: not even close to what was requested.